gallery/visualtests/nxOverflowTooltip.js

```diff
@@ -207,6 +191,6 @@ describe('NxOverflowTooltip', function() {
     await waitAndGetElements(tooltipSelector);
 
     //disabling the region rule to get around the "Some page content is not contained by landmarks" for tooltips
-    await a11yTest(builder => builder.disableRules('region'));
+    a11yTest(builder => builder.disableRules('region'));
```

Comment:
This doesn't seem right.  I think it needs to be 
```suggestion
    await a11yTest(builder => builder.disableRules('region'))();
```